instrumentation/spring/spring-boot-autoconfigure/README.md

```diff
@@ -392,6 +440,13 @@ If an exporter is present in the classpath during runtime and a spring bean of t
 
 <!-- Slf4j Log Correlation  otel.springboot.loggers.slf4j.enabled		true   		org.slf4j.MDC -->
 
+##### Resource Properties
+
+| Feature  | Property                                         | Default Value        | ConditionalOnClass |
+|----------|--------------------------------------------------|----------------------|--------------------|
+| Resource | otel.springboot.resource.enabled                 | false                | OsResource         |
```

Comment:
I think it may be ok to make this default to `true`, similar to the otel autoconfigure module